frontends/web/src/components/devices/bitbox02bootloader/bitbox02bootloader.tsx

```diff
@@ -156,13 +161,15 @@ class BitBox02Bootloader extends Component<Props, State> {
                         </a>
                     </div>
                     <hr/>
-                    <div>
-                        <a
-                            onClick={() => { this.setState({ advancedSettings: !advancedSettings }); }}
-                            style="text-decoration: underline; cursor: pointer;" >
-                            { advancedSettings ? '-' : '+' } Advanced Settings
-                        </a>
-                    </div>
+                    <div className="box slim divide">
+                        <div class="flex flex-row flex-between flex-center">
```

Comment:
this should be `flex flex-row flex-between flex-items-center`

Reply:
Thanks Jeff, fixed.